libqtile/backend/x11/xcbq.py

```diff
@@ -52,7 +52,7 @@
 from libqtile import xkeysyms
 from libqtile.backend.x11.xcursors import Cursors
 from libqtile.log_utils import logger
-from libqtile.utils import hex
+from libqtile.utils import color_hex
```

Comment:
Is that name okay for you?

Reply:
I guess it could be more descriptive like "as_hex" or something but I don't think it's very important at this point. :)